src/windows/webview.c

```diff
@@ -503,21 +506,35 @@ static HMODULE webview_load_dll(void)
 		const WCHAR *path = L"\\EBWebView\\x86\\EmbeddedBrowserWebView.dll";
 	#endif
 
-	if (wcscat_s(dll, MAX_PATH, path) != 0)
+	if (wcscat_s(dll, MTY_PATH_MAX, path) != 0)
 		goto except;
 
-	lib = LoadLibrary(dll);
-	if (!lib)
-		goto except;
+	if (pathw)
+		_snwprintf_s(pathw, MTY_PATH_MAX, _TRUNCATE, L"%s", dll);
 
-	except:
+	ok = true;
 
-	MTY_Free(dll);
+	except:
 
 	if (key)
 		RegCloseKey(key);
 
-	return lib;
+	return ok;
+}
+
+static HMODULE webview_load_dll(void)
+{
+	WCHAR path[MTY_PATH_MAX] = {0};
+
+	// Try system WebView
+	if (webview_dll_path(path, false))
+		return LoadLibrary(path);
+
+	// Try user WebView
+	if (webview_dll_path(path, true))
+		return LoadLibrary(path);
+
+	return NULL;
```

Comment:
Good idea to split out DLL path resolution from DLL loading.

Reply:
It was necessary to fix the bug 🙃

Reply:
Arguably not necessary, nonetheless the right thing to do.